src/lib/content.d.ts

```diff
@@ -252,12 +266,15 @@ interface ProfileAtomBlockElement {
 	role?: RoleType;
 }
 
-interface PullquoteBlockElement {
+interface PullquoteBlockElement extends ThirdPartyEmbeddedContent {
 	_type: 'model.dotcomrendering.pageElements.PullquoteBlockElement';
 	html?: string;
-	role: string;
+	role: RoleType;
```

Comment:
I'd be careful with doing this, it's very possible that we are being sent data that does not confirm to `RoleType` which is why this is a string. I recall a problem where `halfWidth` was being sent as `halfwidth` although that may have just been for RichLinks